_includes/signup.html

```diff
@@ -1,7 +1,5 @@
-<div class="sign-up-container">
-  <a href="https://runnable.com/signup/" class="well well-anchor sign-up-box grid-block col track-sign-up noscroll">
-    <img class="img no-touching" src="{{ site.url }}/images/runnabear-hello.png">
-    <p class="p">Try Runnable! It’s <strong class="weight-strong">free</strong> for 14 days.
-    <div data-target="#sign-up" class="btn btn-sm green js-modal track-sign-up">Sign Up</div>
-  </a>
-</div>
+<a href="https://runnable.com/signup/" class="sign-up-box grid-block col well well-anchor track-sign-up">
```

Comment:
Should you remove the border on the banner too?

![image](https://cloud.githubusercontent.com/assets/10872529/25818549/c7b4e326-33df-11e7-8bbc-be6c5414ba50.png)